Test/auditor/TestAuditor.dfy-ietf.md.expect

```diff
@@ -53,3 +53,7 @@
 ### Member `LoopWithoutBody`
 
 * Definition contains loop with no body. MUST provide a body.
+
+### Member `ConcurrentMethod`
+
+* Declaration has `{:concurrent}` attribute. MUST manually review and test in a concurrent setting.
```

Comment:
Does the auditor have a way of guiding the user to further documentation about a particular class of assumption and mitigation? I feel like this one deserves a bit of text about how Dafny can't (yet) analyze this for you etc.

Reply:
It might make sense to add some text including a link to documentation, for all of the assumption types. I'll look into that.

Reply:
I updated the reference manual to describe all of the forms of assumption that the auditor reports, including a link to rfcs#6 in the bullet for `{:concurrent}`. The generated report currently doesn't include a link to that section of the manual, however.

Reply:
I think that's good enough for now at least, so a user can search the manual for that attribute and get a hit.